app/Models/EventMeetingUserRegistered.php

```diff
@@ -35,15 +35,15 @@ class EventMeetingUserRegistered extends Model
 
     protected $guarded = [];
 
-    public function event(){
+    public function event() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->